src/main/scala/firrtl/proto/FromProto.scala

```diff
@@ -8,7 +8,9 @@ import java.io.{File, FileInputStream, InputStream}
 import collection.JavaConverters._
 import FirrtlProtos._
 import com.google.protobuf.CodedInputStream
-import Firrtl.Statement.{ReadUnderWrite, Formal}
+import Firrtl.Statement.{Formal, ReadUnderWrite}
+
+import scala.collection.mutable
```

Comment:
why mutable here?

Reply:
I will try to change `mutable.Seq` to `scala.collection.Seq`.

Reply:
fixed in https://github.com/freechipsproject/firrtl/pull/1796/commits/4265ed79eff1e5106a1dfab25b30466c17839ae3